### PR TITLE
Feature/core updates

### PR DESCRIPTION
**Pull request by guilhermedecampo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Change Log
```
Comment (grsabreu) [inline CHANGELOG.md]: 👏

File: packages/core/media.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const BASE_FONT_SIZE = 16
+const SIZES = {
+  desktop: 1025,
+  tablet: 900,
+  mobile: 737,
+}
+
+export const media = Object.keys(SIZES).reduce((acc, label) => {
+  const mediaStart = label.includes('desktop') ? 'min' : 'max'
```
Comment (grsabreu) [inline packages/core/media.ts]: includes could be just `===` as this expects a exact match, less noise

File: packages/core/media.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  acc[label] = (args: TemplateStringsArray) => css`
+    @media (${mediaStart}-width: ${SIZES[label] / BASE_FONT_SIZE}em) {
```
Comment (grsabreu) [inline packages/core/media.ts]: BASE_FONT_SIZE shouldn't be 1rem instead to be sure it's the root base?

File: packages/core/media.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+const BASE_FONT_SIZE = 16
+const SIZES = {
+  desktop: 1025,
+  tablet: 900,
+  mobile: 737,
+}
+
+export const media = Object.keys(SIZES).reduce((acc, label) => {
+  const mediaStart = label.includes('desktop') ? 'min' : 'max'
+  acc[label] = (args: TemplateStringsArray) => css`
```
Comment (grsabreu) [inline packages/core/media.ts]: I would just do `return { ...acc, acc[label]: etc }` just for the principle of non-mutation, tho this does no harm in this very context

Comment (guilhermedecampo) [reply packages/core/media.ts]: Hmm indeed. An important change. I actually just got the example from styled and added here quickly.

**Pull request merged**